crates/core/src/contract/storages/redb.rs

```diff
@@ -13,7 +13,8 @@ pub struct ReDb(Database);
 
 impl ReDb {
     pub async fn new(data_dir: &Path) -> Result<Self, redb::Error> {
-        let db_path = data_dir.join("db");
+        let mut db_path = data_dir.join("freenet");
```

Comment:
see what we did in the other open PR https://github.com/freenet/freenet-core/pull/1068
so the solution is constant